### ISSUE TITLE
Regressions: Webfont -> SVG

### ISSUE THREAD

**Issue by sebastianriese** [fsfw-dresden/homepage#45]
- The icons are too high for the header line (you can see this when hovering). This also causes a line-break in the first heading on `index.html`
- The pdf-icon is no longer properly aligned with respect to the text "Druckfassung". Also the other logo is a [pdfreaders.org](http://pdfreaders.org) logo and therefore, in my opinion, preferable (and also available as SVG, but browsers tend to render SVG worse than Inkscape, therefore a prerendered version was used). (I guess we should link pdfreaders.org somwhere, but that's a separate issue).

**Comment by s72785** [comment]
that are two separate things.. should be two issues ...
1. is handled already, was apparently messed by change in stylesheet by someone working on the built system who also mixes is and class selection in css for a single element **
2. sounds like a good idea to use that one, the original one was green but I had no clue where it was from or what license to link.

**Comment by s72785** [comment]
if svg is no looking nice enough a png can be dropped in as base64 as well ;)

**Comment by sebastianriese** [comment]
Yup, I know. I don't know however, whether I like that (the obvious advantage is, that is saves one http request, but that is not that much, considered that the logo will be cached – and consume less cache space as binary file compared to its b64 version). But to ensure coherency with the other images, it should probably included in the css. 

By the way, how did I mess up the layout. I checked and I do not seem to have change the CSS or the html in the merge to `feature/jinja` (only adding staging-related things).

As to the over-specific CSS: So what? (I am honestly interested, not being a web hacker). Isn't it even more extensible that way?

**Comment by s72785** [comment]
my intention was to move all - at least the smaller or probably more often used - graphics to css. but we will see what problems we drop on our feet by doing such.

I was confused as I saw this change before live but not visible in git and adapted to be as to the current built; thats why you dont see the diff there as I had it in branch late. Now also found a solution to circumvent the boxes of icons ditching the headlines.

css: just use id only once (as intended) and then you do not have any need of classes on top just that id only. and it makes the css easier to read as well and probably more compatible, jic.

please keep judging if this is enough or something needs to be done still.

**Comment by sebastianriese** [comment]
I don't use the id any more, that was my first solution, the match is now `div.staging-warning` and the tag only carries a `class="staging-warning"` attribute (I was uncomfortable with using the id-match, as it will not appear at all in the documents that are not staging builds, although I guess browsers accept that without complaining).

I agree we should put the png/svg of the pdfreaders.org logo in the CSS for consistency and then close this issue.

**Comment by sebastianriese** [comment]
I don't see issues with the SVG version of the logo.
